Fix #14: Add new match form

nitink4472 opened the pull request:
<!--
(Thanks for sending a pull request!)
-->

<!--The PR title should start with "Fix #issue_num: " (if applicable), followed by a clear one-line present-tense summary of the changes introduced in the PR. For example: "Fix #issue_num: Introduce the first version of the collection editor." -->

<!-- Add issue number here. If you do not solve the issue entirely, please change the message e.g. "Addresses #IssueNumber -->

Fixes #14

#### Changes proposed in this pull request:

<!-- Changes: Add here what changes were made in this issue and if possible provide links. -->

-

nitink4472 committed:
Update footer.php

footer.php is updated.

nitink4472 committed:
Updated index.php

Index.php has been updated.

nitink4472 committed:
Update common.css

nitink4472 committed:
Update navbar.php

nitink4472 committed:
Update navbar.php

nitink4472 committed:
Update common.js

ambarvm committed:
code cleaning🧹 and minor fixes
- remove repeated cdn links
- remove footer color
- code formatting
- move scroll to top button to footer file

ambarvm committed:
remove repeated script tag

nitink4472 committed:
Update Index.php

Index.php has been updated.

nitink4472 committed:
Update common.js

nitink4472 committed:
Update index.php

supercoww commented:
@nitink4472 remove jQuery and use the javascript DOM APIs wherever jQuery is used. This will reduce the bundle size of the app.

nitink4472 committed:
Update common.js

nitink4472 commented:
> @nitink4472 remove jQuery and use the javascript DOM APIs wherever jQuery is used. This will reduce the bundle size of the app.

@supercoww Just check it now.

ambarvm committed:
🔧Refactor code and fix styling issues
- rename variables and functions
- update scrollToTop for smooth behaviour
- remove redundant materialize init code
- remove redundant link tags
- move new match form to separate page
- add gitignore file

ambarvm committed:
add 🔥firebase initialization code

nitink4472 committed:
Added firebase.config.php
